pom.xml

```diff
@@ -115,7 +115,8 @@
     <module>google-cloud-pubsub</module>
     <module>google-cloud-resourcemanager</module>
     <module>google-cloud-spanner</module>
-    <module>google-cloud-speech</module>
+    <module>google-cloud-speech-v1</module>
+    <module>google-cloud-speech-v1beta1</module>
```

Comment:
I think I would prefer a single artifact that contains both major versions in it. Putting the major version in the artifact doesn't follow the patterns we have been setting up.